src/main/java/org/prebid/server/bidder/HttpBidderRequester.java

```diff
@@ -128,6 +130,17 @@ public HttpBidderRequester(HttpClient httpClient,
                         ? BidderError.Type.timeout
                         : BidderError.Type.generic;
 
+        if (errorType == BidderError.Type.timeout) {
+            if (bidder instanceof TimeoutBidder) {
+                final TimeoutBidder<T> timeoutBidder = (TimeoutBidder<T>) bidder;
+                final HttpRequest<Void> timeoutNotification = timeoutBidder.makeTimeoutNotification(httpRequest);
+                if (timeoutNotification != null) {
+                    httpClient.request(timeoutNotification.getMethod(), timeoutNotification.getUri(),
+                            timeoutNotification.getHeaders(), timeoutNotification.getBody(), NOTIFICATION_TIMEOUT_MS);
+                }
+            }
+        }
```

Comment:
Please extract separate method. 

And extract `timeoutNotification.*` into values.